src/components/proofs/EmailProof.tsx

```diff
@@ -81,12 +81,28 @@ const questionBlock = (open: boolean) =>
 
 const tooltipWrapper = classnames(display('flex'), flex('flex-1'))
 
-export default function () {
-  const [domain, setDomain] = useState('')
-  const [open, setOpen] = useState(false)
+export default function ({
+  domain: defaultDomain,
+  token: defaultToken,
+}: {
+  domain?: string
+  token?: string
+}) {
+  const { emailDomain } = useSnapshot(EmailDomainStore)
+  const [domain, setDomain] = useState(defaultDomain ?? '')
```

Comment:
You're checking `??` in `EmailProofSection` - the only place where we use `<EmailProof />`, I think that you need to:
1) Remove `?` from arguments
2) Remove checks from here, you're passing 100% string